src/style.css

```diff
@@ -851,24 +849,32 @@ blockquote p:after {
   font-weight: 100;
 }
 
-.rule-content h1, .rule-content h2, .rule-content h3, .rule-content h4, .rule-content h5, .rule-content h6, .rule-content .h1, .rule-content .h2, .rule-content .h3, .rule-content .h4, .rule-content .h5, .rule-content .h6 {
-  margin-top: 2rem;
+.rule-content h1, .rule-content .h1 {
+  font-size: 2.5rem;
 }
-
-.rule-content h1 {
-  color: #cc4141;
+.rule-content h2, .rule-content .h2 {
+  font-size: 2rem;
 }
-
-.rule-content h3 {
-  color: #cc4141;
+.rule-content h3, .rule-content .h3 {
+  font-size: 1.75rem;
 }
-
-.rule-content h4 {
-  color: #414141;
+.rule-content h4, .rule-content .h4 {
+  font-size: 1.5rem;
+}
+.rule-content h5, .rule-content .h5 {
+  font-size: 1.25rem;
+}
+.rule-content h6, .rule-content .h6 {
+  font-size: 1rem;
 }
+  
+.rule-content h1, .rule-content h2, .rule-content h3, .rule-content h4, .rule-content h5, .rule-content h6, .rule-content .h1, .rule-content .h2, .rule-content .h3, .rule-content .h4, .rule-content .h5, .rule-content .h6 {
+  margin-top: 2rem;
+}
+  .rule-header-container h1 {margin-top: 1rem;}
 
-.rule-content h4 {
-  word-break: break-word;
+.rule-content h1, .rule-content h2, .rule-content h3 {
+  color: #cc4141;
```

Comment:
Can we try to stick to Tailwind colors?

You can use @apply to use them from the Tailwind config.
https://github.com/SSWConsulting/SSW.Rules/blob/235d745e6eeeb5f95289b54b336a182004075d8c/src/style.css#L100

Reply:
Hey @bradystroud 

See this commit https://github.com/SSWConsulting/SSW.Rules/pull/1078/commits/3c8693032673af5af509641f018ca4f7dbc9e1b4